src/components/page/TwoCol.jsx

```diff
@@ -41,95 +40,18 @@ class TwoCol extends React.PureComponent {
       ...shared,
     };
 
-    const blocksGroups = [
-      {
-        type: 'image',
-        blocks: images,
-      },
-      {
-        type: 'video',
-        blocks: videos,
-      },
-      {
-        type: 'content',
-        blocks: contents,
-      },
-      {
-        type: 'widget',
-        blocks: widgets,
-      },
-      {
-        type: 'table',
-        blocks: tables,
-      },
-      {
-        type: 'checkpoint',
-        blocks: checkpoints,
-      },
-      {
-        type: 'question',
-        blocks: questions,
-      },
-    ];
-
     return (
       <div className="container-flex spaced">
         <div className="col padded col-width-50">
           <div className={leftColGrid}>
             <h1 className={`space-bottom section-title ${gridTitle}`}>
               {title}
             </h1>
-            {/* Top Left */}
-            <BlocksLayout
-              layout={{
-                getRow: 'top',
-                getCol: 'left',
-              }}
-              {...{ blocksGroups, blockShared }}
-            />
-            {/* Middle Left */}
-            <BlocksLayout
-              layout={{
-                getRow: 'middle',
-                getCol: 'left',
-              }}
-              {...{ blocksGroups, blockShared }}
-            />
-            {/* Bottom Left */}
-            <BlocksLayout
-              layout={{
-                getRow: 'bottom',
-                getCol: 'left',
-              }}
-              {...{ blocksGroups, blockShared }}
-            />
+            <BlocksColumn getCol="left" {...{ blocksGroups, blockShared }} />
           </div>
         </div>
         <div className={`col padded col-width-50 col-fixed ${rightColGrid}`}>
-          {/* Top Right */}
-          <BlocksLayout
-            layout={{
-              getRow: 'top',
-              getCol: 'right',
-            }}
-            {...{ blocksGroups, blockShared }}
-          />
-          {/* Middle Right */}
-          <BlocksLayout
-            layout={{
-              getRow: 'middle',
-              getCol: 'right',
-            }}
-            {...{ blocksGroups, blockShared }}
-          />
-          {/* Bottom Right */}
-          <BlocksLayout
-            layout={{
-              getRow: 'bottom',
-              getCol: 'right',
-            }}
-            {...{ blocksGroups, blockShared }}
-          />
+          <BlocksColumn getCol="right" {...{ blocksGroups, blockShared }} />
           {this.isPosEmpty({ col: 'right' }, [
             ...(tables || []),
             ...(images || []),
```

Comment:
I think we actually take out this whole sense of Placeholders now.  And if we do then we don't need tables, images, widgets, etc. as props anymore either, all the way back up to the PageContainer...I think...which would be nice.  Let's breakout this additional change into its of jira ticket/PR in case it does bork something.